org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java

```diff
@@ -53,11 +53,17 @@
 public class ZipActionImpl extends ContainerActionImpl implements ElementAction {
 
 	public ZipActionImpl(ActionContext context, ActionType actionType) {
+		this(context, actionType, false);
```

Comment:
I am not sure we need to keep the old signature. We are not 1.0 yet, so the API can be changed for this impl class.